src/wasm/lyra_utils_wasm.d.ts

```diff
@@ -0,0 +1,15 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+* @param {IntersectScoresInput} arrays
+* @returns {InsersectScoresOutput}
+*/
+export function intersectTokenScores(arrays: IntersectScoresInput): InsersectScoresOutput;
+export interface IntersectScoresInput {
+    data: [string, number][][];
```

Comment:
Just wondering, is there a way to avoid returning an object containing a `data` property? Not a big deal, just wondering